extensions/amp-analytics/0.1/test/test-activity-impl.js

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
You should be able to reuse the existing tests within `test-activity.js` without creating a new test file. sorry about the confusion in naming : (